addons/survey/static/src/js/survey_session_manage.js

```diff
@@ -181,17 +180,22 @@ publicWidget.registry.SurveySessionManage = publicWidget.Widget.extend({
     _onBack: function (ev) {
         ev.preventDefault();
 
+        if (this.isFirstQuestion) {
+            return;
+        }
```

Comment:
I don't agree with that actually.
Go back can be used to go back from the leaderboard screen to the question results, even for the very first question.

I think you should only "return" if it is the first question AND screenToDisplay is 'previousQuestion'.

Reply:
ho my. you're right